types/react-native-scrollable-tab-view/index.d.ts

```diff
@@ -137,6 +137,7 @@ export interface DefaultTabBarProps {
     tabStyle?: ViewStyle;
     renderTab?: RenderTabProperties;
     underlineStyle?: ViewStyle;
+    style?: ViewStyle;
```

Comment:
Please provide why this change is needed and add a test. Thanks

Reply:
I want modify DefaultTabBar component style, This modification is relatively simple.